### PR TITLE
Adding Persian translation for posts of `Bare Bone` Chapter

### PR DESCRIPTION
**Pull request by hamidrezakp**
All of the four posts are translated to Persian.
Posts 01, 02 has been reviewed by us, but the other two need review.

### CODE REVIEW

File: blog/config.toml [hunk 1]
```diff
@@ -11,6 +11,7 @@ languages = [
     { code = "zh-CN" }, # Chinese (simplified)
     { code = "zh-TW" }, # Chinese (tradictional)
     { code = "ja" }, # Japanese
+    { code = "fa-IR" }, # Persian
```
Comment (phil-opp) [inline blog/config.toml:14]: Are there multiple variants of Persian? Otherwise, we could use the more general `fa` language code instead of `fa-IR`.

**Comment by phil-opp** [comment]
Thanks a lot to both of you!

Unfortunately I don't know the language, so a review is difficult for me.

File: blog/config.toml (hunk 1 above)
Comment (hamidrezakp) [reply blog/config.toml:14]: Actually there is only one variant of Persian language. so we can change it to `fa`.

**Comment by hamidrezakp** [comment]
> Thanks a lot to both of you!

Thanks you verry much
There is a lot of Persian speaking people that are reading your blog and want to contribute in translating it to Persian.

By review i meant Persian speakers to review posts 😄 

also i will commit changes in few hours, i am out of home right now.

**Comment by hamidrezakp** [comment]
I have committed the changes but seems there is a dead link in `post-04/index.md`.

**Comment by hamidrezakp** [comment]
Ok, now its ready to be merged :D

**Comment by phil-opp** [comment]
Awesome, thanks a lot for your work!

**Comment by phil-opp** [comment]
Published at https://os.phil-opp.com/fa/

**Pull request merged**